public/v4/apps/fireflyiii-data-importer.yml

```diff
@@ -0,0 +1,51 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: fireflyiii/data-importer:$$cap_fidi_version
+        environment:
+            FIREFLY_III_ACCESS_TOKEN: $$cap_ff_token
+            FIREFLY_III_URL: $$cap_ff_url
+            NORDIGEN_ID: $$cap_nordigen_id
+            NORDIGEN_KEY: $$cap_nordigen_key
+            SPECTRE_APP_ID: $$cap_spectre_app_id
+            SPECTRE_SECRET: $$cap_spectre_secret
+        ports:
+            - $$cap_fidi_port:8080
```

Comment:
You probably don't need this. This is normally needed when you don't have reverse proxy. But with CapRover, your app is available via  `appname.root.domain.com`

Reply:
You're probably right. I'm running my installation locally and ran into port issues in the past. I'll fix this.